File: Ohtap E Commerce Domain Models/Development/PHP/Objects/Campaigns/ICampaignData.php
<?php

namespace Ohtap\ECommerce\DomainModels\Objects\Campaigns;
use Ohtap\ECommerce\DomainModels\Core\Object as  CoreObjects ;


interface ICampaignData extends ICampaignPhotos,ICampaignEndDate,ICampaignStartDate,CoreObjects\IObjectID, CoreObjects\IObjectCode,
    CoreObjects\IObjectDescription, CoreObjects\IObjectName, CoreObjects\IObjectOrder, CoreObjects\IObjectState, CoreObjects\IObjectTitle
{


}

File: Ohtap E Commerce Domain Models/Development/NetFramework/Models Objects/Models Objects/Campaings/ICampaignData.cs

using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using CoreObject = Ohtap.ECommerce.Domain.Models.Core.Object;

namespace Ohtap.ECommerce.Domain.Models.Objects.Campaigns
{
    public interface ICampaignData : CoreObject.IObjectID, CoreObject.IObjectOrder, CoreObject.IObjectState, CoreObject.IObjectCode,CoreObject.IObjectDescription, CoreObject.IObjectName, ICampaignPhotos, ICampaignStartDate, ICampaignEndDate
    {
    }
}
